.github/workflows/update-token.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
Nice, I didn't know about inputs on workflow_dispatch!

So we are also set permission-wise because we just need to give write access to the repo to whomever is allowed do change the link!